app/ide-desktop/lib/client/src/index.ts

```diff
@@ -145,6 +164,7 @@ class App {
                     enableBlinkFeatures: argGroups.chrome.options.enableBlinkFeatures.value,
                     disableBlinkFeatures: argGroups.chrome.options.disableBlinkFeatures.value,
                     spellcheck: false,
+                    webSecurity: false,
```

Comment:
1. No way we should allow for that.
2. Adding so important changes to the code without any explanation is really bad.
3. In all Electron manuas it is always underlined that this should never be disabled. This PR disables it and gives no information about why this change was made.